gridded_etl_tools/dataset_manager.py

```diff
@@ -9,6 +9,7 @@
 import datetime
 import psutil
 
+from .encryption import register_encryption_key
```

Comment:
`encryption.py` should move into the `utils` folder in keeping with how we handle other helper modules.

Reply:
055d4ab

The reason I initially had it the level I did was it contains one function I consider part of the public API (`generate_encryption_key`) and I was trying to keep that closer to the surface. It wasn't a great solution, though. I think it might be helpful, in a future PR, to think about what the public API for the library is and re-export anything  we expect the user to interact with directly in the root of the package and focus our documentation there.